client/web/src/search/results/streaming/StreamingSearchResults.test.tsx

```diff
@@ -0,0 +1,214 @@
+                    location={history.location}
+                    streamSearch={searchSpy}
+                />
+            </BrowserRouter>
+        )
+
+        sinon.assert.calledOnce(searchSpy)
+        sinon.assert.calledWith(
+            searchSpy,
+            'r:golang/oauth2 test f:travis  case:yes',
+            'V2',
+            SearchPatternType.regexp,
+            'test'
+        )
+
+        element.unmount()
+    })
+
+    it('should update patternType if different between URL and context', () => {
+        const history = createBrowserHistory()
+        history.replace({ search: 'q=r:golang/oauth2+test+f:travis&patternType=regexp' })
+
+        const setPatternTypeSpy = sinon.spy()
+
+        const element = mount(
+            <BrowserRouter>
+                <StreamingSearchResults
+                    {...defaultProps}
+                    history={history}
+                    location={history.location}
+                    patternType={SearchPatternType.literal}
+                    setPatternType={setPatternTypeSpy}
+                />
+            </BrowserRouter>
+        )
+
+        sinon.assert.calledOnce(setPatternTypeSpy)
+        sinon.assert.calledWith(setPatternTypeSpy, SearchPatternType.regexp)
+
+        element.unmount()
+    })
+
+    it('should not update patternType if different between URL and context', () => {
+        const history = createBrowserHistory()
+        history.replace({ search: 'q=r:golang/oauth2+test+f:travis&patternType=regexp' })
+
+        const setPatternTypeSpy = sinon.spy()
+
+        const element = mount(
+            <BrowserRouter>
+                <StreamingSearchResults
+                    {...defaultProps}
+                    history={history}
+                    location={history.location}
+                    patternType={SearchPatternType.regexp}
+                    setPatternType={setPatternTypeSpy}
+                />
+            </BrowserRouter>
+        )
+
+        sinon.assert.notCalled(setPatternTypeSpy)
+
+        element.unmount()
+    })
+
+    it('should update caseSensitive if different between URL and context', () => {
+        const history = createBrowserHistory()
+        history.replace({ search: 'q=r:golang/oauth2+test+f:travis case:yes' })
+
+        const setCaseSensitivitySpy = sinon.spy()
+
+        const element = mount(
+            <BrowserRouter>
+                <StreamingSearchResults
+                    {...defaultProps}
+                    history={history}
+                    location={history.location}
+                    caseSensitive={false}
+                    setCaseSensitivity={setCaseSensitivitySpy}
+                />
+            </BrowserRouter>
+        )
+
+        sinon.assert.calledOnce(setCaseSensitivitySpy)
+        sinon.assert.calledWith(setCaseSensitivitySpy, true)
+
+        element.unmount()
+    })
+
+    it('should not update caseSensitive if different between URL and context', () => {
```

Comment:
Same comment as above: caseSensitive is true for both the URL and the prop
```suggestion
    it('should not update caseSensitive if URL and context are the same', () => {
```